cid.go

```diff
@@ -51,15 +51,16 @@ var (
 // the codes described in the authoritative document:
 // https://github.com/multiformats/multicodec/blob/master/table.csv
 const (
-	Raw = 0x55
-
-	DagProtobuf = 0x70
-	DagCBOR     = 0x71
-	Libp2pKey   = 0x72
-
-	GitRaw = 0x78
-
-	DagJOSE               = 0x85
+	// core IPLD
```

Comment:
I don't think I'd differentiate in comments "core IPLD" from other codecs.

That being said I think the groupings are fine as they highlight the ones developers are currently most likely to see in the wild.

Reply:
yeah, although `Libp2pKey` isn't "core IPLD", probably not a big deal though